drivers/can/Kconfig.mcp251xfd

```diff
@@ -58,4 +58,31 @@ config CAN_MAX_FILTER
 	help
 	  Maximum number of filters supported by the can_add_rx_callback() API call.
 
+config CAN_MCP251XFD_PM_DEVICE
+	bool "Enable power management support"
+	default n
+	depends on PM_DEVICE
+	help
+	  Enable power management support for the MCP251XFD CAN controller.
+	  When enabled, the driver implements suspend and resume callbacks
+	  that transition the device to sleep mode on suspend and restore
+	  the previous operational mode on resume.
```

Comment:
This should not be a dedicated, driver-specific Kconfig. Enabling `CONFIG_PM_DEVICE` should be sufficient.